starter/src/index.html

```diff
@@ -55,20 +15,16 @@
 	<!-- Webpack Dev Server reload -->
 	<script src="/webpack-dev-server.js"></script>
 	<% } %>
-
+	
+	<!-- Webpack DLL CSS files -->
+	<!-- The base href is defined through the Webpack configuration and can be customized through the angular.json file -->
 	<!-- add this block at the end of the <head> section -->
 	<% if (htmlWebpackPlugin.options.dllFiles) { %>  <!-- dllFiles will be filled in by the Dll plugin -->
-	<!-- Webpack DLL CSS files -->
+	<!-- necessary for HTML5 History API which adds support for pretty URLs (see app.ts) -->
```

Comment:
The same as my previous comment, should be put before the `<meta charset="utf-8">` element above (on line 4)